core/services/feeds/service.go

```diff
@@ -1086,7 +1086,9 @@ func (s *service) Start(ctx context.Context) error {
 				}
 			}
 		} else {
-			s.connectFeedManager(ctx, mgrs[0], privkey)
+			if mgrs[0].DisabledAt == nil {
```

Comment:
Are we asserting logic based on a side effect? `DisabledAt` reads like it would hold a timestamp. Is there a more direct state variable you can read from? Just from reading, something like `Disabled` or `Enabled` would be more appropriate.

Reply:
Hey patrick, my auto merged kicked in and merged this PR. 

But let me explain, so as you have guessed correctly ,the `DisabledAt` is a timestamp field in feeds manager table that holds a time when user disables a feeds manager, and enabling it will clear this field. This is the only field we have, we did started with a single `Enabled` field, but we ended up with `DisabledAt` to be consistent with other services ([CLO](https://github.com/smartcontractkit/feeds-manager/blob/8e30f86a37bc5ed754ab9d77892c904926b82c26/pkg/entities/node.go#L29)) that we have mainly for consistency.